hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/FileFormatUtilsForFileGroupReader.scala

```diff
@@ -124,4 +124,11 @@ object FileFormatUtilsForFileGroupReader extends SparkAdapterSupport {
       plan
     }
   }
+
+  def createStreamingDataFrame(sqlContext: SQLContext, relation: HadoopFsRelation, requiredSchema: StructType): DataFrame = {
+    val logRel = LogicalRelation(relation, isStreaming = true)
+    val resolvedSchema = logRel.resolve(requiredSchema, sqlContext.sparkSession.sessionState.analyzer.resolver)
+    Dataset.ofRows(sqlContext.sparkSession, applyFiltersToPlan(logRel, requiredSchema, resolvedSchema,
```

Comment:
As a separate follow-up, could you check if the `DataSourceReadOptions.PUSH_DOWN_INCR_FILTERS` is applied in the new stream source read path?  I found that the `HoodieCopyOnWriteIncrementalHadoopFsRelationFactory` does not seem take filters from the this config.